src/gtimelog/timelog.py

```diff
@@ -1134,15 +1131,8 @@ class CSVWriter(object):
     def __init__(self, *args, **kw):
         self._writer = csv.writer(*args, **kw)
 
-    if PY3:  # pragma: PY3
-        def writerow(self, row):
-            self._writer.writerow(row)
-    else:  # pragma: PY2
-        def writerow(self, row):
-            self._writer.writerow([
-                s.encode('UTF-8') if isinstance(s, unicode) else s  # noqa: F821
-                for s in row
-            ])
+    def writerow(self, row):
+        self._writer.writerow(row)
```

Comment:
The entire CSVWriter class can go away now, we can use `csv.writer` directly.

Reply:
Indeed.